packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -201,6 +203,15 @@ class Tooltip extends StatefulWidget {
   /// pointer exits the widget.
   final Duration? showDuration;
 
+  /// The mode that will decide when the tooltip will trigger
```

Comment:
```suggestion
  /// The [TooltipTriggerMode] that will show the tooltip.
```

This should also mention whether or not it can be null, what the default is as well as the ability to set this using the theme.

Reply:
Do we need to specify nullable? we do have null-safety in place for that.

Reply:
We typically do for documentation.